test/vtl.test.ts

```diff
@@ -1,8 +1,7 @@
 import "jest";
-import { $util } from "../src";
-import { AppsyncContext } from "../src";
+import { $util, AppsyncContext } from "../src";
 import { reflect } from "../src/reflect";
-import { returnExpr, appsyncTestCase } from "./util";
+import { appsyncTestCase, returnExpr } from "./util";
 
```

Comment:
What did you run that updated these?

Reply:
I think it was the ESLint VS Code plugin

Reply:
Things I don't run because they eat all of my RAM

Reply:
Download more ram